ports/qt-advanced-docking-system/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "qt-advanced-docking-system",
-  "version-string": "3.6.3",
-  "port-version": 2,
+  "version-string": "3.7.2",
```

```suggestion
  "version": "3.7.2",
```
Same with you previous PR 😄